location/http/handler/location_handler.go

```diff
@@ -0,0 +1,35 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-playground/validator/v10"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+)
+
+type Location struct {
+	Latitude  float32 `json:"latitude" validate:"required,latitude"`
+	Longitude float32 `json:"longitude" validate:"required,longitude"`
+}
+
+func CourierHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
rename HandlerCouriersLocation and add how method structure
func (h *LocationHandler) HandlerCouriersLocation(w nethttp.ResponseWriter, r *nethttp.Request) {